src/coord/src/error.rs

```diff
@@ -243,6 +244,13 @@ impl CoordError {
                 let doc_page = match source_type {
                     RematerializedSourceType::Postgres => "postgres",
                     RematerializedSourceType::S3 => "text-s3",
+                    RematerializedSourceType::PersistedSource => {
+                        // TODO: Make this more helpful once we have documentation for persisted
+                        // sources.
+                        return Some(
+                            "Try creating one materialization and then base more views on top of that.".to_string(),
```

Comment:
maybe something like "A SOURCE being persisted may only be materialized once. Try creating a MATERIALIZED SOURCE, MATERIALIZED VIEW, or MATERIALIZED INDEX."?

Reply:
That first part is already being emitted by other parts, this code here is only responsible for the _hint_. The full output, as of this change is:
```
materialize=> CREATE MATERIALIZED VIEW plain AS
  SELECT a, b, ts
  FROM none_source;
ERROR:  Cannot re-materialize source none_source
DETAIL:  Persisted sources can be materialized by only one set of indexes at a time. The following indexes have already materialized this source:
    plain2_primary_idx
HINT:  Try creating one materialization and then base more views on top of that.
```

Reply:
oh gotcha! my thought was more generally about the "materialization" jargon as opposed to concretely listing what gets you a materialization. ignore if "materialization" is a word our users are already expected to be familiar with

Reply:
Yeah we should definitely generate a canonical doc page for this, if for no other reason than it will be extremely satisfying to delete it when the platform vision is made real.

I would consider being more explicit about the users two options, something like:

```text
Either create one materialization and base more views on that, or create another source to base this materialization upon.
```